packages/terra-site/src/examples/form/examples/Select.jsx

```diff
@@ -7,9 +7,9 @@ import Select from 'terra-form/lib/Select';
 const SelectExamples = () => (
   <form>
     <Select
-      choices={['Puppies', 'Kittens', 'Snappers', 'Bumblers', 'Joeys', 'Microprocessors']}
+      choices={['wow', 'this', 'that']}
```

Comment:
Ah I changed the initialized value to be null instead of undefined so it was skipping my choices/options check. Both props should work to populate the Select so that is backwards compatible. I've fixed it now but I'll re-add tests with the choices to prop to ensure it's working.